src/platform/site-wide/cta-widget/index.js

```diff
@@ -395,7 +396,25 @@ export class CallToActionWidget extends React.Component {
     return this.props.profile.verified;
   };
 
-  openLoginModal = () => this.props.toggleLoginModal(true);
+  logGAEvent = () => {
+    const { appId } = this.props;
+    const toolContexts = [
+      'rx',
+      'messaging',
+      'appointments',
+      'lab-and-test-results',
+      'health-records',
+      'letters',
+    ];
+
+    const eventContext = toolContexts.indexOf(appId) !== -1 ? 'tools' : 'forms';
```

Comment:
I think we should be able to use `Array.prototype.includes` here.

Reply:
Looking at the other apps that use the CTA widget, I'm not sure if we can always assume it's forms in the negative case though; it would apply to something like Claims & Appeals, VIC, etc.